pkg/integrations/manager.go

```diff
@@ -0,0 +1,72 @@
+package integrations
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/redhat-appstudio/tssc-cli/pkg/integration"
+	"github.com/redhat-appstudio/tssc-cli/pkg/k8s"
+)
+
+// Manager represents the actor responsible for all integrations. It centralizes
+// the management of integration instances, keeping a consistent set of
+// integration names.
+type Manager struct {
+	integrations map[string]*integration.Integration // integration handlers
+}
+
+// IntegrationName name of a integration.
+type IntegrationName string
+
+const (
+	ACS            IntegrationName = "acs"
+	Artifactory    IntegrationName = "artifactory"
+	Azure          IntegrationName = "azure"
+	BitBucket      IntegrationName = "bitbucket"
+	GitHubApp      IntegrationName = "github-app"
+	GitLab         IntegrationName = "gitlab"
+	Jenkins        IntegrationName = "jenkins"
+	Nexus          IntegrationName = "nexus"
+	Quay           IntegrationName = "quay"
+	Trustification IntegrationName = "trustification"
+)
+
+const (
```

Comment:
I'm curious, why is this declared here instead of `containerregistry.go`?

Reply:
I won't hold the PR for that, but I'm afraid I don't understand. Let's remember to discuss it on our 1-1.